test/Vagrantfile

```diff
@@ -0,0 +1,48 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+$coreos_channel = "stable"
+$image_version = "current"
+
+$vm_memory = 1024
+$vm_cpus = 1
```

Comment:
Most systems can spare some more mem and CPU, it might make the tests pass quicker. Maybe 2cpus and 2048 mem?